src/list_methods.rb

```diff
@@ -0,0 +1,37 @@
+require_relative './logic'
+
+class ListMethods
+  def self.list_people(people)
+    puts 'There is nobody yet! Kindly add a student or a teacher.' if people.empty?
+    puts
+    people.each_with_index do |person, index|
+      result = "[#{person.class}] ID: #{person.id}, Name: #{person.name}, Age: #{person.age}"
+      result = "#{index}) " + result
+      puts result
+    end
+    sleep 0.75
+  end
+
+  def self.list_books(books)
+    puts 'There are no books yet! Kindly add books.' if books.empty?
+    puts
+    books.each_with_index do |book, index|
+      result = "Title: \"#{book.title}\", Author: #{book.author}"
+      result = " [#{index}] " + result
+      puts result
+    end
+    sleep 0.75
+  end
+
+  def self.list_rentals(people)
+    puts 'No person entry found' if people.empty?
+    puts
+    puts 'Enter person ID: '
+    person_id = gets.chomp.to_i
+    person = Logic.find(people, person_id)
+    puts 'Rentals:'
+    person.rentals.each do |rental|
+      puts "Date: #{rental.date} Book: #{rental.book.title} by #{rental.book.author}"
+    end
```

Comment:
[OPTIONAL] For a better user experience, it would be nicer to handle the situation where the entered id does not exist. When a user inputs an ID that doesn't exist, your web app now displays the following error:

<img width="683" alt="image" src="https://user-images.githubusercontent.com/77190203/220926502-1c20ed68-cb13-4937-b44d-18fab5321143.png">

Reply:
Thank you for the review and feedback. Optional changes implemented in the code. Thanks